docs/testing.md

```diff
@@ -39,20 +39,54 @@ They generally follow TPCH but are not guaranteed to be the same.
 
 ## Unit tests
 
-Unit tests exist in the tests directory. This is unconventional and is done so we can run the tests
-on the final shaded version of the plugin. It also helps with how we collect code coverage.
+Unit tests exist in the [tests](../tests) directory. This is unconventional and is done so we can
+ run the tests on the final shaded version of the plugin. It also helps with how we collect code coverage.
+
+In order to run the unit-tests issue the maven command to run the tests i.e. `mvn test`. 
+
+To run individual tests append `-DwildcardSuites=<comma separated list of wildcard suite names to execute>` to the above command 
```

Comment:
does this work for both scala and java tests? I know the java ones are different in spark

Reply:
wildcardSuites only applies to scalatest.  Java tests need to be filtered via surefire's approach which is `-Dtest=...`

Reply:
right so we have a java test right - TestHashedPriorityQueue.java?  so adding how to run it would be useful and specifying here this works for scala tests.

Reply:
I guess this text is old new one mentions scala tests